src/block/block_ckpt.c

```diff
@@ -447,31 +450,68 @@ __ckpt_add_blkmod_entry(
     /* Make sure we're not going to run past the end of the bitmap */
     WT_ASSERT(session, blk_mod->bitstring.size >= __bitstr_size((uint32_t)blk_mod->nbits));
     WT_ASSERT(session, end_bit < blk_mod->nbits);
-    /* Set all the bits needed to record this offset/length pair. */
-    __bit_nset(blk_mod->bitstring.mem, start_bit, end_bit);
+    /* Change all the bits needed to record this offset/length pair. */
+    if (set)
+        __bit_nset(blk_mod->bitstring.mem, start_bit, end_bit);
+    else {
+        /*
```

Comment:
Hi @sueloverso , the Code Change Report indicates that the tests used for code coverage (which includes `test/csuite/incr_backup/test_incr_backup`) don't reach this block of new code (lines 457 to 482). Are there other tests that that indicate we're covering the new/changed code?

Reply:
Thank you @jeremythorp for this comment. Closer inspection indicates that the change I put in for named checkpoints was too aggressive and was not clearing bits when it should. I am going to fix that and now I need to rerun my format and other testing because you're right that this new code was not getting exercised like I thought.